app/resources/scanned_resources/scanned_resources_controller.rb

```diff
@@ -41,6 +41,12 @@ def manifest
     redirect_to manifest_scanned_resource_path(id: @resource.id.to_s)
   end
 
+  def cached_manifest(resource, auth_token_param)
+    Rails.cache.fetch("#{ManifestKey.for(resource)}/#{auth_token_param}") do
```

Comment:
Could we add a test that illustrates the role of the auth token here? Showing what happens if the same resources is requested with / without. I think it's worth calling attention to beyond just having it in the method.

Reply:
Added a spec.